packages/fxa-profile-server/.nsprc

```diff
@@ -6,13 +6,15 @@
   "comment_1478": "Denial of Service vulnerability.  Need to upgrade to @hapi/@hapi.  See https://github.com/mozilla/fxa/issues/2601",
   "comment_1479": "Prototype Pollution vulnerability.  Need to upgrade to @hapi/@hapi.  See https://github.com/mozilla/fxa/issues/2601",
   "comment_1481": "Denial of Service vulnerability.  Need to upgrade to @hapi/@hapi.  See https://github.com/mozilla/fxa/issues/2601",
+  "comment_1482": "Denial of Service vulnerability.  Need to upgrade to @hapi/@hapi 18+.  See https://github.com/mozilla/fxa/issues/2601",
   "exceptions": [
     "https://npmjs.com/advisories/1168",
     "https://npmjs.com/advisories/1179",
     "https://npmjs.com/advisories/1464",
     "https://npmjs.com/advisories/1472",
     "https://npmjs.com/advisories/1478",
     "https://npmjs.com/advisories/1479",
-    "https://npmjs.com/advisories/1481"
+    "https://npmjs.com/advisories/1481",
+    "https://npmjs.com/advisories/1482"
```

Comment:
this and above to be fixed by https://github.com/mozilla/fxa/issues/4650 later on